### PR TITLE
Dijkstra Algorithm using JS added

### PR DESCRIPTION
**Pull request by hiteshsubnani0128**
Fixes # 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:

- Dijkstra Algorithm using JS
- 

#### Languages Used:
- JavaScript (Node.JS)
- 

#### Files Added:
- Dijkstra_algo.js
-

### CODE REVIEW

**Comment by jainaman224** [comment]
Please mention the issue you are assigned.

**Comment by jainaman224** [comment]
It would be great if you can use 4 space indentation instead of 2.

**Comment by hiteshsubnani0128** [comment]
> It would be great if you can use 4 space indentation instead of 2.

Done sir

**Comment by jainaman224** [comment]
Please mention the issue you are assigned.

File: Dijkstra_Algorithm/Dijkstra_Algo.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
+//dijkstra algo uisng JavaScript
+function solve(graph, s) {
+      var solutions = {};
```
Comment (jainaman224) [inline Dijkstra_Algorithm/Dijkstra_Algo.js]: These are 5 spaces. A small fix.

File: Dijkstra_Algorithm/Dijkstra_Algo.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+              //reference parent
+              parent = solutions[n];
+              nearest = a;
+              dist = d;
+            }
+          }
+        }
+
+        //no more solutions
+      if(dist === Infinity) {
```
Comment (jainaman224) [inline Dijkstra_Algorithm/Dijkstra_Algo.js]: 3 spaces.

File: Dijkstra_Algorithm/Dijkstra_Algo.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+              continue;
+            //choose nearest node with lowest *total* cost
+            var d = adj[a] + ndist;
+            if(d < dist) {
+              //reference parent
+              parent = solutions[n];
+              nearest = a;
+              dist = d;
+            }
+          }
```
Comment (jainaman224) [inline Dijkstra_Algorithm/Dijkstra_Algo.js]: Wrong indentation.

**Comment by MastersAbh** [comment]
@hiteshsubnani0128 Still working on it ?

**Comment by hiteshsubnani0128** [comment]
Hello sir, I was not able understand how to do that, I mean how to provide 4 space indentation.

**Comment by hiteshsubnani0128** [comment]
Can I have some words with you over any other social media platform so that you guide me more about it.

**Comment by MastersAbh** [comment]
Sure, can you contact me on Slack ? I'd be happy to help

**Comment by MastersAbh** [comment]
Hey, reach out if you have any issue. I go by name Abhinav (P) on Slack.

**Comment by MastersAbh** [comment]
Hi @hiteshsubnani0128 still working on it?

**Comment by MastersAbh** [comment]
@rishabhgarg25699 @jainaman224 you can close this PR due to inactivity.

**Comment by rishabhgarg25699** [comment]
@hiteshsubnani0128 @jainaman224 I am closing this issue due to inactivity of PR owner .

**Pull request closed without merging**